api/utils/keys/policy.go

```diff
@@ -62,6 +77,33 @@ func IsRequiredPolicyMet(requiredPolicy, keyPolicy PrivateKeyPolicy) bool {
 	return false
 }
 
+func (p PrivateKeyPolicy) isHardwareKeyVerified() bool {
```

Comment:
This seems to do the same thing as IsHardwareKeyPolicy below. I think I may be missing the reason this was created instead of using the existing method.